team-agreements/working-agreements/readme.md

```diff
@@ -1,15 +1,63 @@
 # Sections of a working agreement
 
+A working agreement is a document or a set of documents that describe how we work together as a team and what our expectations and principles are.
+
+The working agreement created by the team at the beginning of the project, and is stored in the repository so that it is readily available for everyone working on the project.
+
+The following are examples of sections and points that can be part of a working agreement but each team should compose their own, and adjust times etc. to fit their team needs.
+
+## General
+
+- We work as one team towards a common goal and clear scope
+- We make sure everyones voice is heard, listened to
+- We show all team members equal respect
+- We pair-program as often as possible
+- We make sure to spread our expertise and skills in the team, so no single person is relied on for one skill
+- All times below are listed in CET
+
+## Communication
+
+- We communicate all information relevant to the team through the Project Teams channel
```

Comment:
Can we be that specific about tools here? Not all customers have teams. Maybe something like "_communications tool e.g. Microsoft Teams_", would work?

Reply:
Added that communication channels should be set by the team (in the top section)